stacker/src/arena_hashmap.rs

```diff
@@ -235,6 +235,29 @@ impl ArenaHashMap {
         }
     }
 
+    #[inline]
+    /// This will invalidate the hashmaps get and insert methods.
+    /// Only iter() is still valid afterwards
+    ///
+    /// # Safety
+    /// Any call to get or mutate_or_create after this call is undefined behavior.
+    pub unsafe fn iter_mut_keys<F: Fn(&mut [u8])>(&mut self, cb: F) {
```

Comment:
Why do we need to do the remap here?

COuld that be done in a safe not mut way upon serialization? (in serialize postings)

Reply:
| Why do we need to do the remap here?

We need the keys in lexicographical order, for the serialization.

I think I can change it to consume the hashmap and remove the unsafe